blockchain/v2/processor.go

```diff
@@ -53,7 +45,8 @@ type pcStop struct {
 type pcFinished struct {
 	priorityNormal
 	height       int64
-	blocksSynced int64
+	blocksSynced int
+	tdState      tdState.State
```

Comment:
What is tdState?

Reply:
td tdState is the tendermint consensus state: https://github.com/tendermint/tendermint/blob/master/state/state.go#L51-L83